codegenerator/cli/src/rescript_types.rs

```diff
@@ -10,6 +10,12 @@ use std::{collections::HashSet, fmt::Display};
 
 pub struct RescriptTypeDeclMulti(Vec<RescriptTypeDecl>);
 
+#[derive(Debug, PartialEq, Clone)]
+pub enum RescriptSchemaMode {
+    ForDb,
+    ForFieldSelection,
+}
```

Comment:
Like how it looks like with the explicit mode

Reply:
Yeah definitely much easier to understand